### PR TITLE
Maglabs initial setup

### PR DESCRIPTION
**Pull request by binary1230**
Setting some sane defaults for maglabs deployments
renamed things from 'classic' to 'labs' in deployment settings

### CODE REVIEW

File: event-prime.yaml [hunk 1]
```diff
@@ -33,7 +33,6 @@ uber::config::hotel_req_hour: 30
 uber::config::room_deadline: '2016-01-05'
 
 uber::config::groups_enabled: True
-uber::config::kiosk_cc_enabled: True
```
Comment (binary1230) [inline event-prime.yaml]: moved this from prime-specific to being magfest-specific, it's in common.yaml now.

**Comment by binary1230** [comment]
notes:
* dates are all fake I just changed all 2015 to 2016, it's up to the team to figure out the real dates
* a bunch of plugins we want are disabled, this PR is supposed to be the smallest amount of work needed to spin up a new server
* renamed 'classic' to 'labs' here, which affects the deployments

Please review, however, I'm going to merge this now since it doesn't affect anything currently running or our existing prod servers.

**Comment by binary1230** [comment]
CC @jcooter @migetman9

**Comment by EliAndrewC** [comment]
After skimming through this, it all looks sensible.

**Comment by binary1230** [comment]
woo thanks Eli.

**Pull request merged**